pages/components/modal-utils/modal.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Modal</title>
+    <link rel="stylesheet" href="../../../styles/styles.css">
+    <link rel="stylesheet" href="../modal-utils/modal.css">
+    <link rel="icon" href="../../../assets" type="image/icon type" />
+</head>
+<body>
+    <!-- navbar -->
+   
+        <nav class="navbar flex-sb">
+            <ul class="flex-row">
+                <li><img src="../../../assets/logo.jpg" alt="logo" class="mg-l-xs"></li>
+                <li><a href="../../../index.html" class="padding-m">Home</a></li>
+                <li><a href="../avatar/avatar.html" class="padding-m">Docs</a></li>
+            </ul>
+            <ul class="social-btns flex-row">
+                <li><a href="#" class="padding-m">Linkedin</a></li>
+                <li><a href="#" class="padding-m">Gitthub</a></li>
+            </ul>
+        </nav>
+
+        <div class="wrapper">
+            <ul class="sidebar flex-column-sidebar">
+                
+                <a href="../avatar/avatar.html" class="padding-m">Avatar</a>
+                <a href="../alert/alert.html" class="padding-m">Alert</a>
+                <a href="../badge/badge.html" class="padding-m">Badge</a>
+                <a href="../button/button.html" class="padding-m">Button</a>
+                <a href="../card/cards.html" class="padding-m">Card</a>
+                <a href="../image-input/responsive-images.html" class="padding-m">Image</a>
+                <a href="../image-input/input.html" class="padding-m">Input</a>
+                <a href="../lists/list.html" class="padding-m">List</a>
+                <a href="./modal.html" class="padding-m">Modal</a>
+                <a href="./other-utils.html" class="padding-m">Other Utilities</a>
+            </ul>
+      
+        
+            <div class="page-wrapper">
+                <h1>Modals</h1>
+                <p class="mg-t-s">Copy the styles of different modals and use it in your code</p>
+                <div>
+                    <h3 class="mg-t-s">Modal</h3>
```

Comment:
I don't think you should use class names like this mg-t-s , i am net getting any idea what's this is for try to give some meaningfull name.

Reply:
its a short notation for class margin-top-small.